src/__tests__/Search/CompareWithBase.test.tsx

```diff
@@ -13,38 +15,57 @@ import {
   FetchMockSandbox,
 } from '../utils/test-utils';
 
-function renderComponent(isEditable: boolean) {
+function setupTestData() {
   const { testData } = getTestData();
-  const baseRevs = testData.slice(0, 1);
-  const newRevs = testData.slice(1, 3);
-
-  // The "??" operations below are so that Typescript doesn't wonder about the
-  // undefined value later.
-  const baseRepos = baseRevs.map(
-    (item) => repoMap[item.repository_id] ?? 'try',
+  (global.fetch as FetchMockSandbox).get(
+    'begin:https://treeherder.mozilla.org/api/project/try/push/',
```

Comment:
It's not clear to me why we don't need a mock for `https://treeherder.mozilla.org/api/project/mozilla-central/push/`.
Normally, because the URL we use below is for the mozilla-central repository, the fetches should use this repository too.

My guess: the change of the repository dropdown to redux from the URL is done in an effect, so it's asynchronous, and we're not waiting for it to fire. In the future we'll change that, but that's out of scope for this patch.